tools/accuracy_checker/custom_evaluators/mtcnn_evaluator.py

```diff
@@ -291,6 +291,42 @@ def fit_to_input(self, data, layer_name, layout):
 
         return data
 
+    def preprare_model(self, launcher):
```

Comment:
```suggestion
    def prepare_model(self, launcher):
```

Reply:
oh... I see your comment only after merge... I fixed it in separated PR https://github.com/opencv/open_model_zoo/pull/660